Resources/Maps/_NF/Shuttles/akupara.yml

```
@@ -1587,71 +1725,65 @@ entities:
           friction: 0.4
 - proto: ChairOfficeDark
   entities:
-  - uid: 183
+  - uid: 187
     components:
     - type: Transform
       rot: -1.5707963267948966 rad
       pos: 7.5,8.5
       parent: 1
 - proto: ChairPilotSeat
   entities:
-  - uid: 184
-    components:
-    - type: Transform
-      rot: 3.141592653589793 rad
-      pos: 4.5,12.5
-      parent: 1
-  - uid: 185
+  - uid: 706
     components:
     - type: Transform
       rot: 3.141592653589793 rad
-      pos: 5.5,12.5
+      pos: 4.5,14.5
       parent: 1
 - proto: ChemDispenser
   entities:
-  - uid: 186
+  - uid: 190
     components:
     - type: Transform
-      pos: 6.5,9.5
+      pos: 7.5,9.5
       parent: 1
 - proto: ChemistryHotplate
   entities:
-  - uid: 198
+  - uid: 191
     components:
     - type: Transform
-      pos: 7.5,7.5
+      pos: 8.5,7.5
       parent: 1
 - proto: ChemMaster
   entities:
-  - uid: 187
+  - uid: 192
     components:
     - type: Transform
-      pos: 7.5,9.5
+      pos: 8.5,9.5
       parent: 1
 - proto: CigaretteSpent
   entities:
-  - uid: 189
+  - uid: 194
     components:
     - type: Transform
-      parent: 188
+      parent: 193
     - type: Physics
       canCollide: False
-  - uid: 190
+  - uid: 195
     components:
     - type: Transform
-      parent: 188
+      parent: 193
     - type: Physics
       canCollide: False
 - proto: ClosetRadiationSuitFilled
   entities:
-  - uid: 191
+  - uid: 196
     components:
     - type: Transform
       pos: 3.5,-8.5
       parent: 1
-- proto: ClosetWall
+- proto: ClosetWallMaintenanceFilledRandom
```

```suggestion
- proto: ClosetWall
```
No maints lockers on ships, please.